src/app/app.module.ts

```diff
@@ -199,6 +207,7 @@ registerLocaleData(localeZhTw, 'zh-TW');
         PremiumComponent,
         SettingsComponent,
         ShareComponent,
+        SsoComponent,
```

Comment:
Should not need these in here since these are not modals.

Reply:
Fixed